src/api/app/controllers/webui/user_controller.rb

```diff
@@ -9,7 +9,7 @@ class Webui::UserController < Webui::WebuiController
   skip_before_action :check_anonymous, only: [:do_login]
 
   def index
-    @users = User.all_without_nobody
+    @users = User.all_without_nobody.pluck(:login, :email, :state)
```

Comment:
Please use ``@users = User.all_without_nobody.pluck(:id, :login, :email, :state, :realname)`` as the resulting DB query is the same. However, it will instantiate ActiveRecord objects instead of Arrays and you don't need to change the views. 

Furthermore, from performance perspective this is even worse than it was before because when you pass a string into ``user_with_realname_and_icon`` it will fetch again the User (``user = User.find_by_login(user) unless user.is_a? User``). That means you first select all users from the db and then selecting again every single user :(

Sorry, seems like we were not very clear on the car :disappointed: